src/styles/calendar.scss

```diff
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  gap: $calendar__items-gap;
+  flex-wrap: wrap;
+  max-width: calc(
+    #{$calendar-columns} * #{$calendar__item-size}
+    + (#{$calendar-columns} - 1) * #{$calendar__items-gap}
+  );
+  margin: auto;
+  padding-top: 100px;
+
+  &__item {
+    height: $calendar__item-size;
+    min-width: $calendar__item-size;
+    background-color: hsl(0, 0%, 93%);
+    border: 1px solid black;
+    box-sizing: border-box;
+    text-align: center;
+    transition: transform 0.5s;
```

Comment:
It would be great to create a variable for transition-duration as it should be the same per page